### DISCUSSION TITLE
Update 20210121

### DISCUSSION THREAD

**Discussion by ghost** [rern/rAudio#138] [Fixed Issues]
Updated to the latest version 20210121. No problems with that. Working fine.
One smaller issue. Clicking the rAudio logo is not linking to the GitHub page. Instead a big black page with white letters is shown. Starting with 
{
  "player": "mpd",
  "btclient": "false", etc........
 only clicking on the area in whixh the first { is located makes the page covering screen disappear.
Soft and hard reset did not fix this.
regards

**Comment by EarsofTin** [comment]
No big deal but problem with link to Github persists with latest update "rAudio 1 • 20210125"

**Comment by rern** [comment]
The link can be set as a browser favorite/bookmark . The click has been repurposed to another function instead.

**Comment by EarsofTin** [comment]
Well then it's working just exactly perfect.  Thanks Rern.

**Reply by R101G** [reply]
I think this will probably confuse a new user who finds it by accident. Clicking through to the +R github "home" did have a logic to it.

**Comment by EarsofTin** [comment]
I did like the link to github since that is where the discussion takes place now but easy enough to get there on my own.  I do like the quick digest of settings that now appears.  Maybe a hover tag or label would be more explanatory but certainly not critical to the operation.

**Reply by R101G** [reply]
I agree that to us it's not that important, but a newcomer suddenly presented with that page will assume that something has gone seriously wrong. They will probably not even realise what they did.

**Comment by AndyFx12** [comment]
Link to "Discussions" is back with today's update 20210128.
@rern Thanks!
